### ISSUE TITLE
Promote **re-use** classes, alongside conformance classes

### ISSUE THREAD

**Issue by dr-shorthair** [opengeospatial/ogc-modspec#19]
The modular specification was originally designed to make **conformance testing** manageable, by packaging individual requirements and tests into **classes** to allow conformance claims to be made a useful level of granularity. 

However, an opportunity missed was the possibility of focussing on **reusability** of modules, potentially across specifications. The _dependency_ links imply this, but too many OGC specs replicate modules from other specs, rather than re-using them directly. 

Can we increase the emphasis on **reusable modules** somehow?

**Comment by cnreediii** [comment]
@dr-shorthair Yes, agree. Scott and I have been talking about the building block concept and the relationship between requirements class - conformance class - building block. While the TC PnP does have a discussion on building blocks, we discussed having the ModSpec have a more restricted definition. For example, I do not believe that a "standard" is a building block! A requirements class or requirements module and related conformance test classes a conformance test modules are building blocks. This should generate some discussion :-)

**Comment by dr-shorthair** [comment]
I can only find [sources/sections/00-introduction.adoc](https://github.com/opengeospatial/ogc-modspec/blob/main/sources/sections/00-introduction.adoc) which was last revised 2 years ago - is that what you mean?

**Comment by cnreediii** [comment]
https://github.com/opengeospatial/ogc-modspec/blob/reed-initial-edit/sources/sections/00-introduction.adoc
